### DISCUSSION TITLE
Adding missing cursors

### DISCUSSION THREAD

**Discussion by Lioruby** [tailwindlabs/tailwindcss#3176] [Ideas]
I had the no-css writing objective, but I couldn't because there are cursors missing...
What do you think about add default cursor ?

Lioruby

**Comment by adamwathan** [comment]
You can add whatever cursors you need in your Tailwind config file under the “cursor” key in the “theme” section 👍🏻

**Comment by Lioruby** [comment]
Thanks you for your response ! 
No need to add them all in the source code so ? :)
